package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Please update some other necessary values.
You can see at: https://github.com/cslant/git-notifier-ui/blob/fd95ab42bde5af9fe8992adddb6669c1f05b56ee/package.json